File: abc130/abc130_c.cs
// https://atcoder.jp/contests/abc130/submissions/8296675

using System;
using System.Linq;

namespace CSharp
{
    class Program
    {
        static void Main()
        {
            var input = Console.ReadLine().Split(' ').Select(long.Parse).ToArray();
            var W = input[0];
            var H = input[1];
            var x = input[2];
            var y = input[3];

            Console.WriteLine($"{W * H / 2.0} {(2 * x == W && 2 * y == H ? 1 : 0)}");
        }
    }
}

File: abc/abc130/abc130_c.fs
// https://atcoder.jp/contests/abc130/submissions/11959844

let [| W; H; x; y |] = stdin.ReadLine().Split() |> Array.map float

W * H / 2.0 |> printf "%f "
if 2.0 * x = W && 2.0 * y = H then 1
else 0
|> printfn "%d"
